int8 Nothing
---
int8 Total
